alibi/explainers/anchors/language_model_text_sampler.py

```diff
@@ -60,8 +60,10 @@ def __init__(self, model: LanguageModel, perturb_opts: dict, ):
                 self.subwords_mask[vocab[token]] = True
 
         # define head, tail part of the text
-        self.head, self.tail = '', ''  # type: str, str
-        self.head_tokens, self.tail_tokens = [], []  # type: List[str], List[str]
+        self.head: str = ''
```

Comment:
Since this crops up in a few places, we probably need to decide on a convention for handling these single-line multi-definition bits with type annotations. We could do something like:

```python
self.head: str = ""; self.tail: str = ''"
```

Reply:
I'm happy with the above. This [PEP](https://peps.python.org/pep-0526/#global-and-local-variable-annotations) suggest placing the type hints before the multiple assignment like so:

```py
self.head: str
self.tail: str
self.head, self.tail = "", ""
```

Reply:
I might be being silly, but I can't actually see where they discuss multiple assignment in that PEP? Only a bit about tuple unpacking and conditional branches...

In any case, I sort of think what you posted above is more readable than what I suggested?  Although it doesn't save as many lines...

Reply:
@mauicv @jklaise what was the decision with this? Just to leave as separate lines?

Reply:
Didn't seem to be very important, separate lines is nice as corresponds closely to how you might define types in a dataclass.